pandas/tests/indexes/common.py

```diff
@@ -808,6 +808,13 @@ def test_map_dictlike(self, mapper):
         result = index.map(mapper(expected, index))
         tm.assert_index_equal(result, expected)
 
+    def test_map_str(self):
```

Comment:
Decided to write a common test for `index.map(str)` that shared across all index types since it didn't look like we were generically testing something similar.  Can remove this an write and `IntervalIndex` specific test if that'd be preferred.